### PR TITLE
Issue 149 Removes glide dependency in favor of dep

### PR DESCRIPTION
**Pull request by tjj5036**
### Description
Removes dependency on glide in favor of using dep. Not how how much portability this offers (since find is used), but happy to iterate on it.

### Checklist
- [x] Code compiles correctly
- [x] Run ``make fmt`` to fix inconsistent formats
- [x] Run ``make lint`` to get lint errors and fix all of them
- [x] Created tests which fail without the change (if possible)
- [x] All tests passing
- [x] Extended the README / documentation, if necessary

### CODE REVIEW

**Comment by smtakeda** [comment]
Thanks for the clean up! will review the code in the earliest opportunity.

File: gosnowflake.mak [hunk 1]
```diff
@@ -1,32 +1,35 @@
 ## Setup
+SHELL := /bin/bash
+SRC = $(shell find . -type f -name '*.go' -not -path "./vendor/*")
+
 setup:
 	go get golang.org/x/crypto/ocsp
```
Comment (smtakeda) [inline gosnowflake.mak]: can you remove this line as well? This has been redundant, and it is good chance to remove it.

**Comment by smtakeda** [comment]
Can you remove all of `cmd/*/glide.lock`?

File: benchmark/largesetresult/Makefile [hunk 2]
```diff
@@ -1,6 +1,8 @@
 ## Setup
+SHELL := /bin/bash
+SRC = $(shell find . -type f -name '*.go' -not -path "./vendor/*")
+
 setup:
-	go get github.com/Masterminds/glide
```
Comment (smtakeda) [inline benchmark/largesetresult/Makefile]: can you get the latest `dep` here? It seems dep cannot download itself.

All of Travis test have this
```
dep ensure
dep: WARNING: Unknown field in manifest: prune
```

File: gosnowflake.mak (hunk 1 above)
Comment (tjj5036) [reply gosnowflake.mak]: Which line? `go get golang.org/x/crypto/ocsp`?

File: benchmark/largesetresult/Makefile (hunk 2 above)
Comment (tjj5036) [reply benchmark/largesetresult/Makefile]: Absolutely

File: gosnowflake.mak (hunk 1 above)
Comment (smtakeda) [reply gosnowflake.mak]: Yes `go get .../ocsp`. The package should be updated by `dep`

**Comment by tjj5036** [comment]
Is `TestLargeSetResult` a deterministic test?

**Comment by smtakeda** [comment]
Should be... But I see the similar incident before. Anyway that's a separate issue and I'll investigate later. For now, can you restart the test?

**Comment by smtakeda** [comment]
Looks good. Thanks for the clean up!

**Pull request merged**